lib/rubycritic/analysers/helpers/reek.rb

```diff
@@ -1,15 +1,6 @@
 require "reek"
-require "reek/configuration/app_configuration"
-require "ostruct"
 
 module Rubycritic
-
   class Reek < ::Reek::Examiner
```

Comment:
The idea is to keep this subclass around in case we need to re add any customisations to the Reek API, right?